src/transformers/azrael/config.json

```diff
@@ -0,0 +1,4 @@
+{
+  "address": ["0x94D8f036a0fbC216Bb532D33bDF6564157Af0cD7"],
+  "events": ["Rented", "Lent", "CollateralClaimed", "LendingStopped", "Returned"]
```

Comment:
btw, I have realised these are not used anywhere, so we can just remove them. And the address can be sourced from other places. So will probably have to remove the config.json completely (I will do it in the main at some point).